pidx_test.go

```diff
@@ -1,10 +1,13 @@
 package main
 
 import (
+	"errors"
 	"fmt"
 	"net/http"
 	"net/http/httptest"
 	"testing"
+
+	"bou.ke/monkey"
```

Comment:
This dependency has been archived on github. You may need to find a replacement in the future.

Reply:
Do you have a good replacement for monkey patching?